packages/terra-date-picker/tests/nightwatch/components/DatePickerDefault.jsx

```diff
@@ -6,9 +6,301 @@ const locale = document.getElementsByTagName('html')[0].getAttribute('lang');
 
 const DatePickerDefault = () => (
   <Base locale={locale}>
-    <DatePicker
-      name="date-input"
-    />
+    <div>
```

Comment:
Why is this like this?

Reply:
Totally forgot about this!

I had issues using popper, and so I created a large scroll page to test scrolling with popper. This can all be undone.